docs/installation/ui-installation.md

```diff
@@ -10,25 +10,126 @@ Integrate HolmesGPT into your [K9s](https://github.com/derailed/k9s){:target="_b
+    - **macOS**: `~/Library/Application Support/k9s/plugins.yaml` or `~/.k9s/plugins.yaml`
+    - **Windows**: `%APPDATA%/k9s/plugins.yaml`
+
+    Read more about K9s plugins [here](https://k9scli.io/topics/plugins/){:target="_blank"} and check your plugin path [here](https://k9scli.io/topics/config/){:target="_blank"}.
+
+    ```yaml
+    plugins:
+      holmesgpt:
+        shortCut: Shift-H
+        description: Ask HolmesGPT
+        scopes:
+          - all
+        command: bash
+        background: false
+        confirm: false
+        args:
+          - -c
+          - |
+            # Check if we're already using the correct context
+            CURRENT_CONTEXT=$(kubectl config current-context 2>/dev/null || echo "")
+            if [ "$CURRENT_CONTEXT" = "$CONTEXT" ]; then
+              # Already using the correct context, run HolmesGPT directly
+              holmes ask "why is $NAME of $RESOURCE_NAME in -n $NAMESPACE not working as expected"
+            else
+              # Create temporary kubeconfig to avoid changing user's system context
+              # K9s passes $CONTEXT but we need to ensure HolmesGPT uses the same context
+              # without permanently switching the user's kubectl context
+              TEMP_KUBECONFIG=$(mktemp)
+              kubectl config view --raw > $TEMP_KUBECONFIG
+              KUBECONFIG=$TEMP_KUBECONFIG kubectl config use-context $CONTEXT
+              # KUBECONFIG environment variable is passed to holmes and all its child processes
+              KUBECONFIG=$TEMP_KUBECONFIG holmes ask "why is $NAME of $RESOURCE_NAME in -n $NAMESPACE not working as expected"
+              rm -f $TEMP_KUBECONFIG
+            fi
+            echo "Press 'q' to exit"
+            while : ; do
+            read -n 1 k <&1
+            if [[ $k = q ]] ; then
+            break
+            fi
+            done
+    ```
+
+??? note "Advanced Plugin (Shift + Q) - Interactive plugin with custom questions"
+
+
+    Add to your K9s plugins configuration file:
+
+    - **Linux**: `~/.config/k9s/plugins.yaml` or `~/.k9s/plugins.yaml`
+    - **macOS**: `~/Library/Application Support/k9s/plugins.yaml` or `~/.k9s/plugins.yaml`
+    - **Windows**: `%APPDATA%/k9s/plugins.yaml`
+
+    Read more about K9s plugins [here](https://k9scli.io/topics/plugins/){:target="_blank"} and check your plugin path [here](https://k9scli.io/topics/config/){:target="_blank"}.
+
+    ```yaml
+    plugins:
+      custom-holmesgpt:
+        shortCut: Shift-Q
+        description: Custom HolmesGPT Ask
+        scopes:
+          - all
+        command: bash
+        background: false
+        confirm: false
+        args:
+          - -c
+          - |
+            INSTRUCTIONS="# Edit the line below. Lines starting with '#' will be ignored."
+            DEFAULT_ASK_COMMAND="why is $NAME of $RESOURCE_NAME in -n $NAMESPACE not working as expected"
+            QUESTION_FILE=$(mktemp)
+
+            echo "$INSTRUCTIONS" > "$QUESTION_FILE"
```

Comment:
@pavangudiwada can we just drop the user in interactive mode now?

Reply:
After we release a new version, I think we can just run holmes ask and remove a plugin and some config. Right now we would have to change things too. A default question because Holmes won't have enough context about the app(I could be wrong). 
```
holmes ask QUESTION -i 
```